src/main/java/org/folio/innreach/domain/listener/KafkaCirculationEventListener.java

```diff
@@ -52,6 +53,33 @@ public void handleLoanEvents(List<ConsumerRecord<String, DomainEvent<LoanDTO>>>
     });
   }
 
+  @KafkaListener(
+    containerFactory = KAFKA_CONTAINER_FACTORY,
+    id = "${kafka.listener.request.id}",
+    groupId = "${kafka.listener.request.group-id}",
+    topicPattern = "${kafka.listener.request.topic-pattern}",
+    concurrency = "${kafka.listener.request.concurrency}")
+  public void handleRequestStorage(List<ConsumerRecord<String, DomainEvent<RequestDTO>>> consumerRecords) {
+    log.info("Handling circulation Requests events from Kafka [number of events: {}]", consumerRecords.size());
+    var events = consumerRecords.stream()
+      .filter(e -> e.value() != null)
+      .map(this::toDomainEventWithRecordId)
+      .collect(Collectors.toList());
+
+    eventProcessor.process(events, event -> {
+      switch (event.getType()) {
+      case CREATED:
+        //
+        break;
+      case UPDATED:
+        transactionActionService.handleRequestUpdate(event.getData().getNewEntity());
+        // request is associated with an active INN-Reach Item Hold transaction
```

Comment:
it's actually unknown at this point if request is associated with any transaction. Remove this log or move it to the service method

Reply:
Done

Reply:
Deleted